### DISCUSSION TITLE
Rom Preferences prevent launch of roms with the same name in different systems

### DISCUSSION THREAD

**Discussion by Daeymon** [eduardofilo/RG350_adam_image#284] [SimpleMenu topics]
### What happened?

If you have two identically named games in two different systems, and set rom preferences on one of them, the other will no longer launch through Simple Menu. I suspect this is because both games are trying to launch from the same rom preferences file, which is invalid for the game it wasn't created for.

### Device

RG280V

### Version

v2.1

**Comment by Daeymon** [comment]
There was a fix in the Experimental build of SimpleMenu to resolve this issue: [Fixing preferences to go into section folders ](https://github.com/fgl82/simplemenu/commit/70f405bd860ef3e53b45c1ff3c7cdf01ece0a3f3)

I guess it could help if your running your own build of SimpleMenu.

**Comment by eduardofilo** [comment]
I know the issue with rom preferences and the experimental branch of SimpleMenu. In fact the developer passed me a build of that branch with the issue solved which was the one I incorporated in version 1.3 of the image but which I had to remove in the next version because it turned out to be much more unstable than the previous version.

I have thought about taking a look at the SimpleMenu code to try to contribute improvements and solutions to some of the bugs, but I am not a C developer. I also don't consider the Adam image as a development project but as a distribution of the system, emulators and frontend of others. With its advantages, disadvantages and bugs.

**Reply by Daeymon** [reply]
That's fair, feel free to close. I think the developer of SimpleMenu has either moved on from the project or is on hiatus. If he returns, maybe he'll make the experiemental build more stable.

**Reply by eduardofilo** [reply]
I'll pass you the version that the SimpleMenu developer gave me with the experimental branch. As I said before, it is the one that was installed in version 1.3 of the image. Maybe it won't give you too many problems and it will help you with the management of your ROMs.

[SimpleMenu-OD-BETA.opk.zip](https://github.com/eduardofilo/RG350_adam_image/files/9229327/SimpleMenu-OD-BETA.opk.zip)

**Reply by eduardofilo** [reply]
If you install it, remember to delete the `last_state.sav` file and maybe the `favorites.sav` file (in external SD from v2.0).

**Reply by Daeymon** [reply]
Well that worked. Thankyou. I remember the instability issues, clearing out last_state.sav solves it, thought couldn't quite figure out what caused it. Normally happened when I messed around with the section groups.
